modules/i2c/i2cs.h

```diff
@@ -0,0 +1,50 @@
+/** @defgroup i2c I2C
+ * @brief I2C module
+ */
+//@{
+/**
+ * @file
+ * @brief I2Cs definitions
+ */
+#ifndef I2CS_H__
+#define I2CS_H__
+
+#include <avr/io.h>
+#include <avarix/intlvl.h>
+#include <stdint.h>
+#include "i2c_config.h"
+
+// Transaction status defines
```

Comment:
Use `///` for 1-line documentation of elements in Doxygen.
This is equivalent to `/** @brief ... */`.
(Other occurrences below.)